src/org/tensorflow/yolo/view/ClassifierActivity.java

```diff
@@ -136,11 +184,65 @@ private void renderAdditionalInformation(final Canvas canvas) {
             }
         }
 
-        lines.add("Frame: " + previewWidth + "x" + previewHeight);
-        lines.add("View: " + canvas.getWidth() + "x" + canvas.getHeight());
-        lines.add("Rotation: " + sensorOrientation);
-        lines.add("Inference time: " + lastProcessingTimeMs + "ms");
+// shimatani
+        // lines.add("Frame: " + previewWidth + "x" + previewHeight);
+        // lines.add("View: " + canvas.getWidth() + "x" + canvas.getHeight());
+        // lines.add("Rotation: " + sensorOrientation);
+        // lines.add("Inference time: " + lastProcessingTimeMs + "ms");
+
+        Log.d(LOGGING_TAG, String.format("検出名: %s", nowRecognizedClass));
+        lines.add("検出名: " + nowRecognizedClass);
+
+        String msgInf = makeTts(nowRecognizedClass);
+
+        lines.add(msgInf1);
```

Comment:
Why do we need msgInf1 and msgInf2?

Reply:
It is because the descriptive text has become long.
The second line of the display in the lower screen is msg1, the third line is msg2.

[img](https://github.com/keides2/android-yolo-v2/blob/master/sample/%E2%98%85%E7%B2%98%E7%9D%80%E3%83%86%E3%83%BC%E3%83%97%E3%81%A8%E3%83%9C%E3%82%BF%E3%83%B3%E9%9B%BB%E6%B1%A02.png)